app.js

```diff
@@ -22,15 +25,17 @@ app.use((req, res, next) => {
 });
 
 app.use((err, req, res, next) => {
-  console.log(err);
   if (err.msg === "Invalid article id") {
     res.status(400).send({ msg: "Bad request" });
   }
   if (err.msg === "article_id not found") {
     res.status(404).send({ msg: "Article not found" });
   }
   if (err.code === "22P02") {
-    res.status(400).send({ msg: "Bad request" });
+    res.status(400).send({ msg: "Article ID is invalid" });
+  }
+  if (err.msg === "not found") {
+    res.status(404).send({ msg: "Article ID does not exist" });
   }
   next(err);
 });
```

Comment:
You should split the above out into two different error handlers, one for psql codes and one for custom errors/promise rejects 👍